Advanced GenAI guidance

alsavelv opened the pull request:
Tutorial on extracting insights from Teams conversations that was presented at FabCon Vegas'2025. Instrumental for FAQ generation, benchmark curation, and building an adaptive knowledge base for RAG.

alsavelv committed:
FAQ generation notebook

FAQ generation notebook & setup prerequisites - initial checkin

alsavelv committed:
Merge branch 'microsoft:main' into appliedai/quickstartgenai

alsavelv committed:
Create README.md

alsavelv committed:
Add an illustrative image

alsavelv committed:
Update README.md

alsavelv committed:
Update README.md

journeyman-msft commented on docs-samples/data-science/genai-guidance/01-advanced/README.md:
```diff
@@ -0,0 +1,48 @@
+# Build an Adaptive Enterprise Knowledge Base: A Practical Guide
+
+This tutorial provides a  guide to help you build advanced GenAI-powered applications from tribal conversational knowledge. 
```
There's an extra space

> This tutorial provides a

journeyman-msft commented on docs-samples/data-science/genai-guidance/01-advanced/README.md:
```diff
@@ -0,0 +1,48 @@
+After uploading the notebook, you will need to ensure you have the following to insert into a cell for the remainder of the code to work.
+
+## Synthetic Teams Dataset
+This is a synthetic data for demonstrating how to process conversational data from GraphAPI at scale and extract insights, such as automatically build a FAQ for discussion forums, without exposing any real production data of Microsoft. 
```
Microsoft Graph API?

journeyman-msft replied:
Not "production data of Microsoft", more "production data from your company's M365"

journeyman-msft commented on docs-samples/data-science/genai-guidance/01-advanced/README.md:
```diff
@@ -0,0 +1,48 @@
+
+**Themes**: "Chemistry", "Hiking", "Cooking", "Watercolor", "Leadership", "Reading", and"Education"
+
+**Structure**: GraphAPI (https://learn.microsoft.com/en-us/graph/teams-messaging-overview)
```
again, Microsoft Graph API?

journeyman-msft commented on docs-samples/data-science/genai-guidance/01-advanced/README.md:
```diff
@@ -0,0 +1,48 @@
+
+Download: [link](https://aka.ms/funwithteams) 
+
+Alexandra Savelieva, Journey McDowell
```
I think we need to add some CELA legal terms to the README

journeyman-msft commented on docs-samples/data-science/genai-guidance/01-advanced/preprocessing-teams-for-demo-all-data-workshop.ipynb:
Please scrub the metadata from our notebooks before committing.

![image](https://github.com/user-attachments/assets/80613627-73d5-487b-94d4-7ffd9e0a5c16)

![image](https://github.com/user-attachments/assets/b38ee67f-2691-4dbb-b5d2-bf8d6de7986c)

alsavelv replied:
I deliberately kept the output to make the tutorial demo-able directly from the repo, so people get an idea of how the run output would look. This notebook has no secrets (it's leveraging built-in endpoints) so no risk

alsavelv replied:
Scrubbed :)

alsavelv committed:
Update README.md

Updates per Journey's feedback

alsavelv committed:
Follow up-2 on Journey's feedback

Incorporated dataset directly in the tutorial, added a separate notice file about usage, cleaned outputs in the notebook
